### PR TITLE
Move to --no-confirmation for all non interactive modes in rmt-cli

### PR DESCRIPTION
**Pull request by felixsch**
## Description
Instead of using multiple switches for `non interactive`/`no confirmation` in `rmt-cli`, streamline them to `-no-confirmation`.

Note: Since thor is automatically adding positive and negative switches when using a `boolean` switch all these commands
come now with: `--confirmation` and `--no-confirmation`.
Before they where like `--non-interactive` and `-no-non-interactive`.

# How to test this fixes:

```bash
$ = rmt host terminal
# = comment
!  = expectation
```

### Systems
```bash
$ rmt-cli systems purge --help
! --no-confirmation should be available and described

$ rmt-cli systems purge --before 2022-02-02
! Should ask you if you want to delete systems
# cancel purging command (ctrl+c)

$ rmt-cli systems purge --no-confirmation --before 2022-02-02
! Should purge the systems without user interaction
```
---

### Repos
```bash
$ rmt-cli repos clean --help
! --no-confirmation should be available and described

# If the following command does not show any data to be clean, disable a product (pro tip: Take something light to disable like SLES root product to not have to remirror lots of data later)
$ rmt-cli repos clean
! Should ask you if you want to continue with cleaning
# cancel cleaning command (ctrl+c)

$ rmt-cli repos clean --no-confirmation
! Should clean the repository data
```

### CODE REVIEW

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/241632/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests. (Merged b5059362c6c603cf6860b7500eb7a07049c082e8 into ed7c9a254224656d333d9e01309c93f56dca4a5a)

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/241756/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests. (Merged 5d1667a348c0312c52587831b31d8d7a3667a419 into 8776d78d74db14b124a14650baeed30f44b00cca)

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/241775/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests. (Merged 5d1667a348c0312c52587831b31d8d7a3667a419 into 8776d78d74db14b124a14650baeed30f44b00cca)

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/241924/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests.
Failed fast in project rmt on step docker-compose-build-images.
Failed fast in project rmt on step docker-compose-run-tests. (Merged b7e2f5f19ac06a1515e93c10317aeb570279887b into f365152aba8ffdcb8ebba4cf7f2d7137ab42d53b)

File: MANUAL.md [hunk 1]
```diff
@@ -41,10 +41,10 @@ You can install and run this wizard like this:
 
     `rmt-cli systems remove SCC_e740f34145b84523a184ace764d0d597`
     
-  * `rmt-cli systems purge [--non-interactive] [--before date]`:
+  * `rmt-cli systems purge [--non-confirmation] [--before date]`:
```
Comment (josegomezr) [inline MANUAL.md:44]: I think there's an extra `n` here: `--noN-confirmation` ?

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/242070/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests.
Failed fast in project rmt on step docker-compose-build-images.
Failed fast in project rmt on step docker-compose-run-tests. (Merged b7e2f5f19ac06a1515e93c10317aeb570279887b into cf82866246db34eebe5383a0107a3a7973db7101)

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/242071/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests.
Failed fast in project rmt on step docker-compose-run-tests. (Merged c1aa651f5d185723065d757e9dcbe7f0d256d116 into cf82866246db34eebe5383a0107a3a7973db7101)

File: MANUAL.md [hunk 2]
```diff
@@ -41,10 +41,10 @@ You can install and run this wizard like this:
 
     `rmt-cli systems remove SCC_e740f34145b84523a184ace764d0d597`
     
-  * `rmt-cli systems purge [--non-interactive] [--before date]`:
+  * `rmt-cli systems purge [--no-confirmation] [--before date]`:
     Removes inactive systems.
     
-    Use the `--non-interactive` flag so the command does not ask you for
+    Use the `--non-confirmation` flag so the command does not ask you for
```
Comment (josegomezr) [inline MANUAL.md:47]:
```suggestion
    Use the `--no-confirmation` flag so the command does not ask you for
```

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/242085/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged c1aa651f5d185723065d757e9dcbe7f0d256d116 into cf82866246db34eebe5383a0107a3a7973db7101)

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/242871/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests. (Merged 2915c44dbc9eb98744fce400f515536f731b4873 into cf82866246db34eebe5383a0107a3a7973db7101)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/242884/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 2915c44dbc9eb98744fce400f515536f731b4873 into cf82866246db34eebe5383a0107a3a7973db7101)

**Review by josegomezr — APPROVED** [review]
LGTM

**Pull request merged**